spring-boot-modules/spring-caching/pom.xml

```diff
@@ -12,7 +12,7 @@
         <groupId>com.baeldung</groupId>
         <artifactId>parent-boot-2</artifactId>
         <version>0.0.1-SNAPSHOT</version>
-        <relativePath>../parent-boot-2</relativePath>
+        <relativePath>../../parent-boot-2</relativePath>
```

Comment:
let's make the actual physical module spring-boot-modules as its parent, since spring-boot-modules is dependendent further on parent-boot-2 it should work fine.

Reply:
You mean to leave as it is here: `../parent-boot-2/pom.xml`?

Reply:
No, instead of making parent-boot-2 as the parent, make spring-boot-modules as the parent. Spring-boot-modules further declares parent-boot-2 as its parent so we should be ok

Reply:
Done, I did the changes suggested.